docs/source/configuration.rst

```diff
@@ -41,10 +41,10 @@ Consider:
 
 .. code-block:: shell
 
-   tartufo --config myconfig.toml scan-local-repo .
+   tartufo --config myconfig.toml scan-local-repo <target_directory>
 
 ``tartufo`` will look for ``myconfig.toml`` in the current directory, and then look for
-either ``tartufo.toml`` or (if not found) ``pyproject.toml`` in the current
+either ``tartufo.toml`` or (if not found) ``pyproject.toml`` in the target
 directory because that is the target of the scan. Directives in, say,
```

Comment:
```suggestion
either ``tartufo.toml`` or (if not found) ``pyproject.toml`` in the ``target_directory``
because that is the target of the scan. Directives in, say,
```